src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/advice/ExceptionMapper.java

```diff
@@ -60,6 +61,12 @@
         return errorDetailsResponseEntity(ex, NOT_FOUND, EMPTY_RESULT_DATA_ACCESS.getErrorMessage());
     }
 
+    @ExceptionHandler(ConstraintViolationException.class)
```

Comment:
Do we require this?

Reply:
I think once you do the above changes, it will not throw

Reply:
Addressed here: https://github.com/hmcts/rd-caseworker-ref-api/pull/232/commits/c202fa64288a40583cdd76c4211b8477434266d3

Reply:
thanks